org.eclipse.richbeans.widgets/src/org/eclipse/richbeans/widgets/scalebox/NumberBox.java

```diff
@@ -1112,11 +1116,11 @@ public void setFieldOveride(String fieldOveride) {
 		this.fieldOveride = fieldOveride;
 	}
 	
-	public IBeanController getBeanService() {
+	public IBeanController<?> getBeanService() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove usage of generic wildcard type. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1452)